courses/machine_learning/deepdive2/image_understanding/solutions/segmentation.ipynb

```diff
@@ -1772,7 +1772,9 @@
         "\n",
```

Comment:
<ol><li>Check the "Learning Objectives" font size.   </li><li>Remove "You will learn"</li><li>Make the learning objectives "single-space" between the lines.</li></ol><p> </p><p>Thx.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1456/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
- Updated the font size of Learning Objectives.
- Removed "You will learn" instruction.
- Made the learning objectives "single-space" between the lines.

@gstripling  Please review and merge.